src/bridge/views/Login.js

```diff
@@ -0,0 +1,131 @@
+import { Nothing, Just } from 'folktale/maybe';
+import React from 'react';
+import * as azimuth from 'azimuth-js';
+
+import { H1, Row, Col } from '../components/Base';
+import HorizontalSelector from '../components/HorizontalSelector';
+
+import Ticket from './Login/Ticket';
+import Mnemonic from './Login/Mnemonic';
+import Advanced from './Login/Advanced';
+
+import * as need from '../lib/need';
+import { compose } from '../lib/lib';
+import { ROUTE_NAMES } from '../lib/routeNames';
+
+import { withHistory } from '../store/history';
+import { withNetwork } from '../store/network';
+import { withWallet } from '../store/wallet';
+import { withPointCursor } from '../store/pointCursor';
+
+const TABS = {
+  TICKET: Symbol('TICKET'),
+  MNEMONIC: Symbol('MNEMONIC'),
+  ADVANCED: Symbol('ADVANCED'),
+};
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentTab: TABS.TICKET,
+      wallet: Nothing(),
```

Comment:
do you think it's possible to use the global wallet state setter/getter instead of storing it locally and then committing it to the global state once the user presses continue?

Reply:
like how mnemonic uses `setWallet`

Reply:
I think that's what we already do, this line can just be purged.

Reply:
there are a few `setState({ wallet })` calls in the other components as well